setup.py

```diff
@@ -99,7 +96,4 @@ def read_file(fname):
     url='https://github.com/jazzband/django-analytical',
     download_url='https://github.com/jazzband/django-analytical/archive/master.zip',
     cmdclass=cmdclass,
-    install_requires=[
-        'Django>=1.7.0',
-    ],
```

Comment:
Why not update the dependency version?

Reply:
Sometimes people install packages like django-analytical before Django. Only to find out later that "for some reason" a "wrong Django version" was installed. It's not worth that pain.

We should assume Python developers are adults who can read READMEs or the docs if something breaks.